utils/streams.js

```diff
@@ -0,0 +1,156 @@
+import commander from 'commander';
+import through from 'through2';
+import fs from 'fs';
+import path from 'path';
+import { promisify } from 'util';
+import csvjson from 'csvjson';
+// import https from 'https';
+const readFileAsync = promisify(fs.readFile);
+const readDirAsync = promisify(fs.readdir);
+
+function streams() {
+    commander.option('-a, --action <string>', 'action name taken')
+        .option('-f, --file <string>', 'file will read')
+        .option('-p, --path <string>', 'path for css folder')
+        .parse(process.argv);
+    executeTheAction();
+}
+function executeTheAction() {
+    if (process.argv.length < 3) {
+        commander.help();
+    } else if (process.argv[2] == '-h' || process.argv[2] == '--help') {
+        commander.help();       
+    } else {
+        if (!!commander.action) {
+            switch (commander.action) {
+                case 'reverse':
+                    reverse();
+                    break;
+                case 'transform':
+                    transform();
+                    break;
+                case 'outputFile':
+                    outputFile(commander.file);
+                    break;
+                case 'convertFromFile':
+                    convertFromFile(commander.file);
+                    break;
+                case 'convertToFile':
+                    convertToFile(commander.file);
+                    break;
+                case 'cssBundler':
+                    cssBundler(commander.path);
+                    break;
+                default:
+                    console.log('Err!! Invalid arguement passed. Please choose correct action!!!');
+                    break;
+            }
+        } else {
+            console.log('Err!! No argument passed. Please choose correct action!!!');
+            process.exit();
+        }
+    }
+}
+
+function reverse() {
+    process.stdin.on('data', function (data) {
+        var splitString = data.toString().split("").reverse().join("");
+        process.stdout.write(splitString);
+    });
+}
+
+//Transform stream to uppercase
+function transform() {
+    let transToUpperCase = function (data, encoding, next) {
```

Comment:
Please use `const` instead of `let` since this variable won't be changed.

Reply:
Updated code to use const.